politeiawww/mail/limiter.go

```diff
@@ -0,0 +1,159 @@
+// Copyright (c) 2020-2021 The Decred developers
+// Use of this source code is governed by an ISC
+// license that can be found in the LICENSE file.
+
+package mail
+
+import (
+	"fmt"
+	"time"
+
+	"github.com/decred/politeia/politeiawww/user"
+	"github.com/google/uuid"
+)
+
+// Limiter is a wrapper around Mailer for implementing rate limiting functionality.
```

Comment:
There are other places where this rule is violated (see a screenshot below for example).

I thought it is more of an advisory thing, rather than a hard limit (to communicate a hard limit a linter might be useful), so I've tried to stick to it, but didn't exactly follow this rule everywhere.

I've adjusted my code in that respect now (assuming this is related to comments mostly).

![image](https://user-images.githubusercontent.com/7031569/120214079-a3fc9d80-c23c-11eb-86b9-0963a425965f.png)